llvm/test/CodeGen/NVPTX/kernel-param-align.ll

```diff
@@ -0,0 +1,34 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_72 2>&1 | FileCheck %s
+; RUN: %if ptxas %{ llc < %s -march=nvptx64 -mcpu=sm_72 | %ptxas-verify %}
+
+%struct.Large = type { [16 x double] }
+
+; CHECK: .param .u64 .ptr .global .align 16 func_align_param_0,
```

Comment:
You should also have `CHECK-LABEL: <function name>`

Reply:
Added the CHECK-LABEL.